student.rb

```diff
@@ -1,15 +1,12 @@
-class Student
+require_relative 'school_class'
+
+class Student < School
```

Comment:
A student is a (kind of) school? Really?  Make sure your naming makes sense by trying out sentences like this. (I'd go with `Person` for the common superclass for `Student`, `Teacher`, and `ApprenticeTeacher`)